src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export type SortOrder = 'asc'|'desc';
+
+function getAverageGrade(grades:number[]):number {
+  const sumOfGrades = grades.reduce((sum, grade) => sum + grade, 0);
+  const gradesCount = grades.length;
+
+  return sumOfGrades / gradesCount;
```

Comment:
I believe you don't need this variables
```suggestion
  return  grades.reduce((acc, cur) => acc + cur, 0) / grades.length;
```